PowerEditor/installer/APIs/lua.xml

```diff
@@ -1220,24 +1237,24 @@ captures from the pattern; otherwise it returns nil. If pattern specifies no cap
 the whole match is returned. A third, optional numerical argument init specifies where to 
 start the search; its default value is 1 and can be negative. 
         ">
-                <Param name="s" />
-                <Param name="pattern" />
-                <Param name="[, init]" />
+                <Param name="s"/>
+                <Param name="pattern"/>
+                <Param name="[, init]"/>
```

Comment:
It's not necessary to remove all the white space.
Reformatting is not allowed, as I mentioned:
https://github.com/notepad-plus-plus/notepad-plus-plus/pull/12367#issuecomment-1314492403

Reply:
So do I need to fix the small whitespace additions as well?